module/VuFind/src/VuFind/Log/Handler/Office365Handler.php

```diff
@@ -22,47 +22,48 @@
  *
  * @category VuFind
  * @package  Error_Logging
- * @author   Demian Katz <demian.katz@villanova.edu>
+ * @author   Sambhav Pokharel <sambhav.pokharel@gmail.com>
```

Comment:
Looks like you've changed the author to yourself here, but left me in the block above the class. My suggestion would be to put both `@author` tags in both places. I wrote the original version and you made significant contributions to the new version, so it makes sense for us both to have credit, and I generally try to keep the annotation blocks consistent between the header comment and the class comment.